app/controllers/users/invitations_controller.rb

```diff
@@ -188,6 +190,14 @@ def generate_notification(user, target_user, role, team)
       end
     end
 
+    def check_captcha_for_invite
+      if Rails.configuration.x.enable_recaptcha
+        unless verify_recaptcha
+          render json: { recaptcha_error: t('invite_users.errors.recaptcha') }
```

Comment:
would it make sense to send a status code with this (i.e. 400)?

Reply:
👍 Yeap